vector/src/main/java/im/vector/app/features/rageshake/BugReporter.kt

```diff
@@ -507,17 +501,8 @@ class BugReporter @Inject constructor(
      */
     fun openBugReportScreen(activity: FragmentActivity, reportType: ReportType = ReportType.BUG_REPORT, withScreenshot: Boolean = true) {
         screenshot = takeScreenshot(activity)
-<<<<<<< HEAD
-        activeSessionHolder.getSafeActiveSession()?.let {
-            it.logDbUsageInfo()
-            it.cryptoService().logDbUsageInfo()
-        }
-
-        activity.startActivity(BugReportActivity.intent(activity, reportType, withScreenshot))
-=======
         matrix.debugService().logDbUsageInfo()
-        activity.startActivity(BugReportActivity.intent(activity, reportType))
```

Comment:
Why withScreenshot is not used in Element ?

Reply:
is not used in Element, @Florian14 had add it 
<img width="1632" alt="Capture d’écran 2022-07-26 à 14 42 58" src="https://user-images.githubusercontent.com/15031750/181008704-889e5103-2310-4468-95fd-19f81cd1ad84.png">

Reply:
should we keep it ? @yostyle @giomfo

Reply:
yes keep it. we need to have the same behaviour when you click to report a bug in homescreen menu